RELEASE.md

```diff
@@ -19,11 +31,11 @@
 * Make `kedro micropkg package` accept `--verbose`.
 
 ## Documentation changes
-* Added documentation for developing a Kedro project using a Databricks workspace.
-
-## Breaking changes to the API
+* Significant improvements to the documentation that covers working with Databricks and Kedro, including a new page for workspace-only development, and a guide to choosing the best workflow for your use case.
 
 ## Upcoming deprecations for Kedro 0.19.0
+* All instances of `<Name>DataSet` have been renamed to `<Name>Dataset`. Using `<Name>DataSet`
+will emit a `DeprecationWarning` and will not be available `0.19.0`.
```

Comment:
```suggestion
* Renamed `CachedDataSet`, `LambdaDataSet`, `IncrementalDataSet`, `MemoryDataSet` and `PartitionedDataSet` to `CachedDataset`, `LambdaDataset`, `IncrementalDataset`, `MemoryDataset` and `PartitionedDataset`, respectively.
 * Renamed `DataSetError`, `DataSetAlreadyExistsError` and `DataSetNotFoundError` to `DatasetError`, `DatasetAlreadyExistsError`, and `DatasetNotFoundError`, respectively.
```
I would recommend to be a bit more explicit, because we don't want people thinking `CSVDataSet`, etc. have been renamed yet. Furthermore, we decided to hold off on the `AbstractDataSet` rename for now, too, so it's not just stuff outside core not being renamed. Above notes extracted from #2735.

Reply:
Or maybe we can create a little table. Let me see.